rfc/src/rfcs/0001-mir-linker.md

```diff
@@ -203,7 +204,7 @@ See the table bellow for the breakdown of time (in seconds) taken for each major
 the harness verification:
 
 
-| Stage                     | MIR Linker | Alternative |
+| Stage                     | MIR Linker | Alternative 1 |
```

Comment:
No alternative 2 numbers?

Reply:
No, I could try to collect from my old branches, but I don't think it's worth it at this point. If I remember correctly, for Alternative 2, the compilation time was closer to the MIR Linker, but everything else was closer to Alternative 1.